Dockerfile

```diff
@@ -1 +1,22 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_cloudconfig \
+      MAINTAINER="Armedia LLC"
+ARG RESOURCE_PATH="target"
+ARG CLOUD_CONFIG_VERSION=2021.03
+WORKDIR /app
+RUN useradd --create-home --user-group arkcase \
+        && mkdir /app/tmp \
+        && chown -R arkcase:arkcase /app 
+
+USER arkcase
+#COPY the application war files
+COPY --chown=arkcase ${RESOURCE_PATH}/config-server-${CLOUD_CONFIG_VERSION}-SNAPSHOT.jar /app/config-server.jar 
+COPY --chown=arkcase ${RESOURCE_PATH}/start.sh /app/start.sh
```

Comment:
please combine these lines

Reply:
I think a wrapper script should be provided that would (1) fetch the config-server jar file and then (2) run the `docker build` command.

Reply:
That would work.  Should we store the username/password in SSM parameter store and fetch them from that wrapper script?

Reply:
@JonHolman I think that's a good idea

Reply:
@JonHolman I think I am missing something here... I thought we said to store the artifacts in the S3 bucket? Or is it only for third-party artifacts and we store our own artifacts on fileshare.armedia.com?

Reply:
I thought we were storing all artifacts in that s3 bucket.  The script I'm talking about is get-artifacts.sh to retrieve the artifacts from s3 prior to docker build.

Reply:
@JonHolman I still don't understand your comment about sftp...

Reply:
@fabricetriboix shoot, sorry.  That was supposed to say "now", not "not".  I was confused until you said sftp, then I went back to see what I said.  We are NOT using sftp for anything.  Sorry.

Reply:
@JonHolman All right, that makes sense now!